packages/wonder-blocks-popover/src/components/__tests__/popover.test.tsx

```diff
@@ -582,6 +582,41 @@ describe("Popover", () => {
                 }),
             ).toBeInTheDocument();
         });
+
+        it("should correctly describe the popover content core's arial label", async () => {
```

Comment:
Added a unit test for the popover content core's aria label check in the `popover.test` file - hope that works @catandthemachines @jeresig